popup_windows.js

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable no-multi-str */
+/* eslint-disable no-plusplus */
+/* eslint-disable quotes */
+
+const container = document.createElement('div');
+const title = document.createElement('div');
+const aside = document.createElement('aside');
+const openProject = document.querySelectorAll('.see-project');
+const closeProject = document.createElement('button');
+const sct2 = document.querySelector('#stc2');
+
+const image = {
+  imgDesktop: './images/photos/project_img.png',
+  imgMobile: './images/photos/project_img_smal.png',
+};
+
+const text = {
+  txtDesktop: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the\
+  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and\
+  scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry\
+  . Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum\
+  is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard\
+  dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.",
+
+  txtMobile: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.\
+  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown\
+  printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy\
+  text of the printing and typesetting  ever since the 1500s, when an unknown printer took a\
+  galley of type veris lapoa todoe.",
+
+  html: 'html',
+  css: 'css',
+  ror: 'Ruby on Rails',
+  github: 'Github',
+};
+
+/* Basic script */
+
+closeProject.innerHTML = '&times;';
+closeProject.classList.add('close-project');
+
+const h1 = document.createElement('h1');
+h1.innerHTML = 'Multi - Post stories';
```

Comment:
- [ ] The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.** (I can see that you are rendering from your JS file but the HTML code is also present and when I removed it, the text didn't appear, meaning that the JS code is not working as expected, please take a look into this and remove all the static HTML code)